cmd/otel-allocator/main.go

```diff
@@ -21,7 +24,15 @@ import (
 )
 
 var (
-	setupLog = ctrl.Log.WithName("setup")
+	setupLog     = ctrl.Log.WithName("setup")
+	httpDuration = promauto.NewHistogramVec(prometheus.HistogramOpts{
+		Name: "allocator_http_duration_seconds",
+		Help: "Duration of HTTP requests.",
```

Comment:
From the code I can see that received http calls are meant here, but this is not clear to the user.

Reply:
I can update the help to make that more clear, thank you.